src/main/java/org/cactoos/scalar/Folded.java

```diff
@@ -98,6 +99,19 @@ public Folded(
         this.function = fold;
     }
 
+    /**
+     * Secondary vararg ctor.
```

Comment:
@izrik as I see almost in the whole project all constructors have the same description `Ctor.`  (767 cases in fact), except some rare cases. So I think it's better to do the same here because you just repeat type of arguments and don't describe some unexpected behaver or other important information.